assets/variant-list.css

```diff
@@ -256,6 +257,17 @@ variant-list-remove-button .icon-remove {
   display: block;
 }
 
+.variant-list-error {
```

Comment:
Some screensizes the alignment is a bit off but I fixed that in the variantlist PR. Do have the latest changes? Want to ensure we are not introducing this
https://screenshot.click/30-05-a3yyt-dtyzg.png

Reply:
Updated - https://github.com/Shopify/dawn/pull/2754/commits/3581fc3a3835934ae225fbb3f706cd5cb0bbb88e